base/ltkeys.dtx

```diff
@@ -73,6 +73,9 @@
 %    \item \texttt{.code} --- execute arbitrary code
 %     \item \texttt{.if} --- sets a \TeX{} \cs{if...} switch
 %     \item \texttt{.ifnot} --- sets an inverted \TeX{} \cs{if...} switch
+%     \item \texttt{.scope} --- defines whether when the option is for a class
+%       it should be treated as \texttt{local} (the standard setting) or
+%       \texttt{global} (read by packages from the global list)
```

Comment:
Suggestions welcome: I tried to re-write but wasn't any happier

Reply:
An attempt

```suggestion
%     \item \texttt{.scope} --- defines whether the (class) option should
%       be ignored (\texttt{local}, the standard setting) or read by
%       packages (\texttt{global})
```

Reply:
BTW `clsguide.tex` needs the identical doc of `.scope` property too.

Reply:
would possibly be better with commas `, whether when the option is for a class, `  but I think it woudn't hurt to be more verbose:

--- For class options, this specifies whether the option should be treated as \texttt{local} (the standard setting) or         \texttt{global} (read by packages from the global list). For package options this property has no effect.

or  .... "is ignored"  rather than "no effect" ?

Reply:
OK, I've gone with a version of @davidcarlisle's text